math/mathcore/test/testMixMax.cxx

```diff
@@ -0,0 +1,47 @@
+#include "TRandomGen.h"
+#include "TH1.h"
+#include "TStopwatch.h"
+#include "TFitResult.h"
+#include <iostream>
+
+bool testMixMax(int nbin = 1.E6 ) {
+
+
+   //int nbin = 1.E8; 
+   TH1F h1("h1","h1",nbin, 0, 1);
+   TStopwatch w; 
+
+   TRandomMixMax r; 
```

Comment:
I would use a seed in order to make the test exactly reproducible.

Reply:
The test is reproducible, because by default is used a defined seed (e.g. seed=1).

The idea of this test is to reproduce a problem on Ubuntu, then really testing the generator

Reply:
It's great that by default the seed is 1! But isn't this the opposite of TRandom, where the random seed is "random" by default?

Reply:
No, by default in TRandom the seed is always a pre-defined value. 
You need to provide seed=0  to have a random generated seed